Modules/Core/Common/test/itkNeighborhoodOperatorTest.cxx

```diff
@@ -77,45 +78,82 @@ println(const char * c)
 int
```

Comment:
Why not remove now unused `println`?

Reply:
Left it behind inadvertently. Removed in 8449d1a. Thanks.